spinalcordtoolbox/scripts/sct_compute_flow.py

```diff
@@ -16,13 +16,13 @@
 
 def get_parser():
     parser = SCTArgumentParser(
-        description=(
-            "Compute velocity from the MRI phase image for velocity encoding (VENC) sequences."
-            "\nMore details in: https://mriquestions.com/what-is-venc.html"
-            "\n"
-            "\nFurther features are planned for this script. Please refer to this issue for more info:"
-            "\n  - https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4298"
-        )
+        description=("""
```

Comment:
Consider escaping the initial newline:
```suggestion
        description=("""\
```